EliteDangerous/JournalEvents/JournalScan.cs

```diff
@@ -480,28 +483,33 @@ public string DisplayString(int indent = 0, bool includefront = true , MaterialC
             if (HasRings)
             {
                 scanText.Append("\n");
-                if (IsStar)
+                for (var r = 0; r < Rings.Length; r++)
                 {
-                    scanText.AppendFormat("Belt{0}".Tx(this), Rings.Count() == 1 ? ":" : "s:");
-                    for (int i = 0; i < Rings.Length; i++)
+                    // If it's name ends with "Belt"...
+                    if (Rings[r].Name.EndsWith("Belt"))
                     {
-                        if (Rings[i].MassMT > (oneMoon_MT / 10000))
+                        HasBelts = true;
+
+                        // ...than it's a belt!
+                        scanText.AppendFormat("Belt{0}".Tx(this), Rings.Count() == 1 ? ":" : "s:");
```

Comment:
Probably need the Belt(s) and Ring(s) to have separate translated singular and plural forms.

Reply:
Yeah, good point...